src/vs/workbench/parts/debug/electron-browser/electronDebugActions.ts

```diff
@@ -34,6 +34,23 @@ export class CopyValueAction extends Action {
 	}
 }
 
+export class CopyPropertyPathAction extends Action {
+	static readonly ID = 'workbench.debug.viewlet.action.copyPropertyPath';
+	static LABEL = nls.localize('copyPropertyPath', "Copy Property Path");
+
+	constructor(id: string, label: string, private value: any) {
+		super(id, label, 'debug-action copy-property-path');
```

Comment:
No need to pass the css class since this action does not have that, simply pass `undefined` as the third argument